nupic/encoders/scalar.capnp

```diff
@@ -0,0 +1,14 @@
+@0xfa7d16f86048a6e4;
+
+struct ScalarEncoderProto {
+  # ScalarEncoder() constructor signature
```

Comment:
I'm not sure that this comment is necessary. I'd rather have the schema explain the parameters rather than rely on the code implementations.